src/app/header/header-profile/header-profile.component.html

```diff
@@ -1,9 +1,57 @@
 <div class="profile" [class.active]="isCartFill">
-  <a href="#">
+  <span class="profile_user-button" (click)="toggleUserWindow()">
     <img src="assets/ic-actions-user.svg" alt="">
-  </a>
-  <a routerLink="/cart" class="profile_cart {{count ? 'profile_cart--fill' : ''}}">
+  </span>
+  <div class="profile_user-overlay" *ngIf="showUserWindow" (click)="toggleUserWindow()"></div>
+  <div class="profile_user" *ngIf="showUserWindow">
+    <div>
+      Hello{{user.name ? ', ' + user.name : '!'}}
+    </div>
+    <div (click)="!user.id ? toggleModal() : logOut()" class="profile_log">{{!user.id ? 'Log in' : 'Log out'}}</div>
```

Comment:
Please create two buttons and use *ngIf depending on the user id.

Reply:
fixed